case/scad/source/bottom.scad

```diff
@@ -1,7 +1,11 @@
 module
 sample()
 {
-    import(file = "dao-choc-ble-bottom.svg");
+    if(add_pinky) {
```

Comment:
I suggest a space between the `if` and the parenthesis.
I had been using https://www.npmjs.com/package/openscad-format

Reply:
fixed, thank you!